src/time/wasm.rs

```diff
@@ -0,0 +1,23 @@
+#[cfg(target_family = "wasm")]
```

Comment:
the cfg can be put in src/time/mod.rs over the mod wasm instead of having it in this file